src/Core/View.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: Scene
+ * Description: La Scene est l'instance principale du client. Elle est le chef orchestre de l'application.
+ */
+
+/* global window, requestAnimationFrame */
+
+import Camera from '../Renderer/Camera';
+import MainLoop from './MainLoop';
+import c3DEngine from '../Renderer/c3DEngine';
+import { STRATEGY_MIN_NETWORK_TRAFFIC } from './Layer/LayerUpdateStrategy';
+import Scheduler from './Scheduler/Scheduler';
+import WMTS_Provider from './Scheduler/Providers/WMTS_Provider';
+import WMS_Provider from './Scheduler/Providers/WMS_Provider';
+import TileProvider from './Scheduler/Providers/TileProvider';
+import Debug from '../../utils/debug/Debug';
+
+/**
+ * Constructs an Itowns Scene instance
+ *
+ * @param {string} crs - The default CRS of Three.js coordinates. Should be a cartesian CRS.
+ * @param {Coordinates} positionCamera - The initial position of the camera
+ * @param {DOMElement} viewerDiv - Where to instanciate the Three.js scene in the DOM
+ * @param {boolean} debugMode - activate debug mode
+ * @param {boolean} glDebug - debug gl code
+ * @constructor
+ */
+ /* TODO:
+ * - remove debug boolean, replace by if __DEBUG__ and checkboxes in debug UI
+ * - Scene (and subobjects) should be instanciable several times.
+ */
+function View(crs, viewerDiv, mainLoop) {
+    this.referenceCrs = crs;
+
+    this.mainLoop = mainLoop || new MainLoop(new Scheduler(), new c3DEngine(viewerDiv));
+
+    this.camera = new Camera(
+        viewerDiv.clientWidth,
+        viewerDiv.clientHeight);
+
+    this._layers = [];
+
+    if (__DEBUG__) {
+        this._debug = new Debug(this, viewerDiv);
```

Comment:
Nope and Debug is a plain function, not a constructor so  I need to this this... thanks.